### ISSUE TITLE
tedious + azure v12 connectionError/ELOGIN

### ISSUE THREAD

**Issue by spanditcaa** [pekim/tedious#307]
I am having an issue with tedious 1.11.2 and SQL Azure v12 on iojs 2.5.  The error message we get back is different from what is being reported in #272 - 
````
name: ConnectionError
message: Database 'ABC' on server 'aaa1aaaaa' is not currently available.  Please retry the connection later.  If the problem persists, contact customer support, and provide them the session tracing ID of 'CDF3AEE3-4E70-4DAF-85BA-311AE76BEB29'.
code: ELOGIN
````
In case there are any microsoft folks around, additional tracing IDs we've seen include:
````
C6B4392B-08FE-48DC-B058-E2D8A4D52D16
D0CEA175-3F84-44CB-B02E-793CDE938E88
118A46D8-03B8-4409-9849-7F3B3C3425CE
````
It has happened several times a day since our v12 upgrade.  When this does occur, I notice that all instances of our API appear to lose connection simultaneously.  As with #272 if we restart the affected instances everything reconnects and works normally (for 6-12 hours).

Thanks

**Comment by spanditcaa** [comment]
Working with Microsoft, we've believe this is the same port/token issue and should resolve with https://github.com/pekim/tedious/pull/269.

**Comment by arthurschreiber** [comment]
Hey Sanjay, this is great, thanks. I'll take a look at this asap. I'm currently holding off releasing a new version of tedious because I want to get #303 solved beforehand.

Would a new version in the 1.11.x release line with this fix be good enough for you for now?

**Comment by arthurschreiber** [comment]
Fixed by #309 and #310. Will be released in `1.11.5` and `1.12.3`.
